packages/playwright-core/src/server/injected/consoleApi.ts

```diff
@@ -85,7 +85,11 @@ class ConsoleAPI {
       inspect: (selector: string) => this._inspect(selector),
       selector: (element: Element) => this._selector(element),
       generateLocator: (element: Element, language?: Language) => this._generateLocator(element, language),
-      ariaSnapshot: (element?: Element) => this._injectedScript.ariaSnapshot(element || this._injectedScript.document.body),
+      ariaSnapshot: (element?: Element) => {
+        const snapshot = this._injectedScript.ariaSnapshot(element || this._injectedScript.document.body);
+        // eslint-disable-next-line no-console
+        console.log(snapshot);
```

Comment:
We don't print to console in other methods, why not just return the value as before?

Reply:
It is multi-line and I need to console.log to preserve \n there.